polymetis/polymetis/python/polysim/grpc_sim_client.py

```diff
@@ -122,9 +126,25 @@ def __init__(
         self.i = 0
         self.interval_log = []
 
+        self.current_pos_image = None
+        ###self.sio = socketio.Client()
+        ###self.sio.connect('http://172.23.42.96:5000')
+        #self.client = "socket.socket()"
+        #client.connect((TCP_IP, TCP_PORT))
+
     def __del__(self):
         """Close connection in destructor"""
-        self.channel.close()
+        self.channel.close() 
+
+    @Pyro4.expose
+    def get_image(self):
+        arr = self.env.get_current_pos_image()
+        return arr
+
+    @Pyro4.expose
+    def get_sim_state(self):
+        arr = 1
+        return arr
```

Comment:
Not sure what this method does

Reply:
Sorry, this was a debug code that I forgot to remove.